doc/cephfs/snap-schedule.rst

```diff
@@ -38,6 +38,9 @@ below). By default
 the start time is last midnight. So when a snapshot schedule with repeat
 interval `1h` is added at 13:50
 with the default start time, the first snapshot will be taken at 14:00.
+The default time zone of the start time is assumed to be in UTC if there's no
+time zone mentioned in the start time string. If a time zone other than UTC is
```

Comment:
example of a timezone mentioned? what format?

Reply:
Good point!

Reply:
If [ISO_8601](https://en.wikipedia.org/wiki/ISO_8601) time, we should say so.

The first sentence above is a bit stilted too, and I think the   Given the context, I suggest:

"The time zone is assumed to be UTC if none is explicitly included in the string.  An explicit time zone will be mapped to UTC at execution

Example: `Tue Feb  8 10:59:09 CET`
"

If the format is what I think it is.

Reply:
I think the latest push addresses both our comments